app/partials/header.html

```diff
@@ -1,8 +1,41 @@
-<div class="header">
-  <ul class="nav nav-pills pull-right">
-    <li class="active"><a href="#">Home</a></li>
-    <li><a href="#">About</a></li>
-    <li><a href="#">Contact</a></li>
-  </ul>
-  <h3 class="text-muted">PageTraining</h3>
+<br>
+<div class="row top">
+  <div class="col-lg-2">
+    <img src="partials/images/companyname.png"/>
+  </div>
+  <div class="col-lg-1 col-lg-offset-2">
+    <form>
+      <input type="search" name="search" class="search">
+    </form>
+  </div>
+  <div class="col-lg-1 col-lg-offset-6">
+    <img src="partials/images/counter.png"/>
+  </div>
 </div>
+<br>
+<div class="row" id="menu">
```

Comment:
Take a look at bootstrap classes and structure for menus 
http://getbootstrap.com/components/#navbar-default
http://getbootstrap.com/components/#navbar-fixed-top